src/less/menu/ds6/menu.less

```diff
@@ -2,5 +2,5 @@
 
 @import "../../less/ds6/variables.less";
 @import "../../less/ds6/mixins.less";
-@import "../../dropdown/ds6/dropdown.less";
+@import "../../mixins/dropdown/ds4/dropdown-mixins.less";
```

Comment:
Ruh roh. Just spotted this. `ds4` instead of `ds6`. Will fix with direct commit to 8.0.0 branch.